socorro/external/postgresql/crash_migration_app.py

```diff
@@ -42,66 +41,5 @@ def _get_processed_crash_transaction(self, connection, crash_id):
             raise CrashIDNotFound(crash_id)
 
 
-class CrashMigrationApp(RawAndProcessedCopierApp):
-    app_name = 'crash_migration'
-    app_version = '1.0'
-    app_description = __doc__
-
-    required_config = Namespace()
-    required_config.add_option(
-        'no_dumps',
-        doc="don't copy binary dumps",
-        default=True
-    )
-
-    @staticmethod
-    def get_application_defaults():
-        return {
-            'source.crashstorage_class': DumbPostgreSQLCrashStorage,
-            'destination.crashstorage_class':
-                'socorro.external.es.crashstorage.'
-                'ESCrashStorageNoStackwalkerOutput',
-        }
-
-    def _create_iter(self):
-        connection = self.source.database.connection()
-        sql = 'select uuid from raw_crashes;'
-        return execute_query_iter(connection, sql)
-
-    def _transform(self, crash_id):
-        try:
-            raw_crash = self.source.get_raw_crash(crash_id)
-            if self.config.no_dumps:
-                dumps = {}
-            else:
-                dumps =  self.source.get_raw_dumps(crash_id)
-            processed_crash = self.source.get_unredacted_processed(
-                crash_id
-            )
-        except CrashIDNotFound:
-            self.config.logger.warning(
-                'cannot find crash %s',
-                crash_id,
-            )
-            return
-        except Exception:
-            self.config.logger.warning(
-                'error loading crash %s',
-                crash_id,
-                exc_info=True
-            )
-            return
-
-        if 'uuid' not in raw_crash:
-            raw_crash.uuid = crash_id
-        self.destination.save_raw_and_processed(
-            raw_crash,
-            dumps,
-            processed_crash,
-            crash_id
-        )
-        self.config.logger.info('saved - %s', crash_id)
-
-
 if __name__ == '__main__':
     main(CrashMigrationApp)
```

Comment:
This needs to be removed as well. Actually, this entire file should be removed, since the remaining ``DumbPostgreSQLCrashStorage`` class is only used in here.